plugins/videobasedtracker/CMakeLists.txt

```diff
@@ -67,6 +67,8 @@ endif()
 
 if(NOT WIN32)
     find_package(libuvc REQUIRED)
+    find_package(PkgConfig)
```

Comment:
We should have a find module for libusb1 already, shouldn't need to use just PkgConfig.